src/Stripe.net/Entities/PaymentIntents/PaymentIntentTransferData.cs

```diff
@@ -29,5 +27,9 @@ internal object InternalDestination
             }
         }
         #endregion
+
+        [Obsolete("Use ApplicationFeeAmount instead")]
```

Comment:
The class doesn't have an `ApplicationFeeAmount` property. Can you change the message to `"Use PaymentIntent.ApplicationFeeAmount"`?